### ISSUE TITLE
imported from 'svelte/internal'

### ISSUE THREAD

**Issue by Sable-20** [svelteuidev/svelteui#507]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

imported from 'svelte/internal' causing failure to render during SSR

### In which browser(s) did the problem occur?

Firefox

### Steps To Reproduce

use SSR and then use SvelteUIProvider

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by Plumliil** [comment]
When I encountered the same problem after starting the project using vite

**Comment by drewbitt** [comment]
Yes, until this library is updated, it is not able to be used in Svelte 5+.

**Comment by BeeMargarida** [comment]
We are working on a patch for it :saluting_face:

**Comment by g-wozniak** [comment]
Hello, do you know roughly the estimation for this issue to be fixed? I'd love to use `svelteui` in my project as it's my favourite but this is sadly a blocker :(

**Comment by BeeMargarida** [comment]
> Hello, do you know roughly the estimation for this issue to be fixed? I'd love to use `svelteui` in my project as it's my favourite but this is sadly a blocker :(

Sorry, I cant give a prediction right now. Perhaps this [patch](https://github.com/svelteuidev/svelteui/issues/491#issuecomment-2506873532) that iva2k created might help in the meantime

**Comment by BeeMargarida** [comment]
> when will the patch for svelte 5 be released

We are working on it, you can see the progress is the migration/svelte-5 branch. I can't give a timeline on when it will be ready

**Comment by pazfelipe** [comment]
Hey everione, how's it going? Any updates about the release for sveltekit 5? I've been using svelte for few days and it's amazing. I'd like to continue using this framework and integrate svelteui.

**Comment by j4w8n** [comment]
Any closer here? I'd like to keep using this library, as the dx is better than others I've looked into.

**Comment by BeeMargarida** [comment]
> Any closer here? I'd like to keep using this library, as the dx is better than others I've looked into.

We are in the final steps, just need to fix some minor bugs and update the docs with the changes made

**Comment by ericschmar** [comment]
> > Any closer here? I'd like to keep using this library, as the dx is better than others I've looked into.
> 
> We are in the final steps, just need to fix some minor bugs and update the docs with the changes made

Can you please provide an updated timeline? I appreciate all the work you do, and look forward to upgrading!

**Comment by benbucksch** [comment]
Hey, first off, thanks for svelteui.

The currently latest version of svelteui breaks with Svelte 5 and doesn't even compile. It appears that these are rather minor problems, like avoiding a handful imports from "svelte/internal" and replacing them with something else. That seems like a project for a few hours, not years. Would it be possible to release a svelteui 0.16 with only those changes that keep svelteui usable with both svelte 3 and svelte 5.x, at the same time?

As it is, this problem here is preventing the upgrade of our application to Svelte 5.
